examples/whylogs_data_profiling/config.yaml

```diff
@@ -0,0 +1,18 @@
+name: data_profiling_pipeline
```

Comment:
for other examples you mentioned in the README the option to run with `-c config`, why not here?